Update manual.md (#100)
cardonabits/haxo-hw: docs/manual/manual.md

 <a name="playing"></a>

### Playing
Here are some useful links to the haxophone fingering chart and some other fingering charts:

> * [https://www.sebulli.com/haxo/notemap/](https://www.sebulli.com/haxo/notemap/)
> * [https://www.windtabmusic.com/free-area-tenor-sax ](https://www.windtabmusic.com/free-area-tenor-sax)

Reviewer: ```suggestion
* [https://www.sebulli.com/haxo/notemap/](https://www.sebulli.com/haxo/notemap/)
* [https://haxo-notemap.nn.r.appspot.com/](https://haxo-notemap.nn.r.appspot.com/)
* [https://www.windtabmusic.com/free-area-tenor-sax ](https://www.windtabmusic.com/free-area-tenor-sax)
```

    ```
    scp notemap.json pi@haxophone.local:/home/pi
    ```

>    and then on the haxophone copy it to the correct location via sudo cp:

Reviewer: ```suggestion
   and then, on the haxophone, copy it to the correct location via `sudo cp`:
```

 <a name="troubleshooting-faq"></a>

### Troubleshooting / FAQ

> 1. Which Raspberry Pi should I use? Pi Zero is quite fine, but the Zero W or Pi 3 or Pi 4 work, too.

Reviewer: ```suggestion
1. Which Raspberry Pi should I use? At this moment, the Pi Zero 2 is the best Pi for the Haxophone.  It is small, fast, supports MIDI out, and it is relatively inexpensive.  The Pi Zero (or the Zero W) is even cheaper, and plays fine, but will be slow if you want to use it to compile your custom haxophone firmware.  The Pi 3 and Pi 4 work, are good for development but you will not have MIDI out, and will add extra weight.
```
